### ISSUE TITLE
Taxa are not being recognized in Planteome browser

### ISSUE THREAD

**Issue by cooperl09** [Planteome/planteome-core#4]
For example look at any gene or annotation page:
http://planteome.org/amigo/cgi-bin/crop_amigo/gp-details.cgi?gp=GR_QTL:AQFE076&session_id=2996amigo1443573689

They appear to be entered in the file correctly, so namemunger file may need to be updated.

**Comment by elserj** [comment]
Fixed.  The commands that generate the taxon names from the ncbi taxon dump file was adding extra lines that shouldn't have been there.  Not sure why, but seems to be a problem in the association file, not the commands themselves.  Anyway, I manually created the file and uploaded it the data to the MySQL DB.  Fixed now, but have to check if it happens again.  I don't think it will as we have fixed/modified most of the association files recently.  AmiGO2 uses a totally different mechanism to get the taxons loaded anyway (ncbi_taxonomy.owl file), so I don't think this will be a problem once we update.
